python/cugraph/cugraph/tests/dask/test_mg_renumber.py

```diff
@@ -176,3 +176,51 @@ def test_dask_pagerank(dask_client):
             err = err + 1
     print("Mismatches:", err)
     assert err == 0
+
+
+@pytest.mark.skipif(
+    is_single_gpu(), reason="skipping MG testing on Single GPU system"
+)
+@pytest.mark.parametrize("renumber", [False])
+def test_digraph_renumber_false(renumber, dask_client):
+    gc.collect()
```

Comment:
I think these should be removed from each test in this file and replaced with a single setup function, as was done [here](https://github.com/rapidsai/cugraph/blob/2770c87e033c6d6c5c03bfa75d16fb755105c633/python/cugraph/cugraph/tests/test_hits.py#L28).

Reply:
Right, I forgot about it